packages/gitbook-docs/src/markdoc/attributes.ts

```diff
@@ -0,0 +1,33 @@
+import { Config } from '@markdoc/markdoc';
+
+const convertAssetsPath = (assetsPrefix: string, src: string) =>
+  src.replace(
+    new RegExp('^.*(.gitbook/assets/)'),
+    `${assetsPrefix}/.gitbook/assets/`
+  );
+
+const convertLinkPath = (linkPrefix: string, href: string) =>
+  !href.startsWith('http')
+    ? `${linkPrefix}/${href
+        .replace('/README.md', '')
+        .replace('README.md', '')
+        .replace('.md', '')}`
+    : href;
```

Comment:
What do you think to add doc and unit tests on these methods?

Reply:
They should already be covered by [these tests](https://github.com/pagopa/developer-portal/pull/178/files#diff-da1aec090fdbd8c5af03fea302092bf6172f42a1ee6d21479ad41a40a1e57f80R34-R50). The `SrcAttr` (which uses `convertAssetsPath `) is not yet used (it will be by parseBody). I'm going to remove it.

Reply:
I've removed useless code and improved tests 👍